config/environments/production.rb

```diff
@@ -49,7 +49,7 @@
   config.log_level = :debug
 
   # Skip http-to-https redirect for the default health check endpoint.
-  # config.ssl_options = { redirect: { exclude: ->(request) { request.path == "/up" } } }
+  config.ssl_options = { redirect: { exclude: ->(request) { request.path == '/health' } } }
```

Comment:
We need to skip this? Im not sure we should be accessing Limber on http anymore.